### ISSUE TITLE
Weekly Quest - games played challenge not upticking

### ISSUE THREAD

**Issue by SwaghettiYolonaise11** [ValveSoftware/Dota2-Gameplay#3104]
### Description

Hi, I am trying to complete week twos challenges for the battle pass but, while everything else is getting counted, played matches is not ticking up each time I play a game. All other challenges are being upticked just not the played matches. I have tried deactivating the week, restarting, verifying the game file, etc. and nothing. I keep missing taking a screenshot but I can show in the screenshots that I've played plenty more games then what I've been credited for.

### Screenshots

![Screenshot 2022-09-15 115223](https://user-images.githubusercontent.com/113569185/190298492-adda94af-a8e7-4555-a5f4-9d5cf81c7786.png)
![Screenshot 2022-09-15 115310](https://user-images.githubusercontent.com/113569185/190298506-0878982e-67cb-4830-a165-5b4e891ab8db.png)
![Screenshot 2022-09-15 115322](https://user-images.githubusercontent.com/113569185/190298509-ea679b71-73e4-4df1-9fe9-bbb58b9233be.png)
![Screenshot 2022-09-15 115333](https://user-images.githubusercontent.com/113569185/190298512-1ca4bf1f-9c4d-4bfe-8fe4-11811fb7255c.png

**Comment by ashluver89** [comment]
I have the same problem! I do hope they'll provide a fix soon.
![week2](https://user-images.githubusercontent.com/113569921/190300384-a3bbe6ec-1091-4d91-a9c5-67150f8d15c1.png)
![turbo1](https://user-images.githubusercontent.com/113569921/190300399-f880f733-dbf2-4c1c-84af-643b7c592c20.png)

**Comment by ISBadWolf** [comment]
#3083

**Comment by Picnic-Bar** [comment]
Same here.... my number of games should be completed - i have been stuck on 7 games for 2 days

Also See Issues #3097 #3086 #3083 they are all the same issue

**Comment by Green138b** [comment]
The challenge ends in 2 hours...and they still did not fix it...so we can't take last battle chest..

**Comment by ISBadWolf** [comment]
> The challenge ends in 2 hours...and they still did not fix it...so we can't take last battle chest..

You can always reselect the quest week later. I hope the fix would involve retroactively progressing the quest but it may not be possible.

**Comment by UncleComfortable** [comment]
Yeah, this is especially annoying as week 2 is a bonus reward week, and getting to 15 games has been really hard as a somewhat casual player. I hope it's retroactive, I'll be so annoyed if it isn't.

**Comment by TheUnholyWrath** [comment]
You can close this issue.
